src/employeeProfiles.js

```diff
@@ -57,6 +67,33 @@ const GetActiveContract = () => {
   return activeContract;
 };
 
+const GetVacationDetail = () => {
+  const employeeId = useGetRecordId();
+  const { data: vacations } = useGetManyReference("vacations", {
+    target: "employeeId",
+    id: employeeId,
+  });
+
+  vacations?.forEach((v) => (v.remainingDays = v.credit - v.debit));
+  return vacations;
+};
+
+const GetAvailableDays = () => {
+  let totalCredit = 0;
+  let totalDebit = 0;
+  let vacations = GetVacationDetail();
+  let availableDays = 0;
+
+  vacations?.forEach((v) => {
+    if (parseInt(v.year) <= new Date().getFullYear()) {
```

Comment:
why this check?

Reply:
i think we shouldn't consider the vacations credit of the next year